### DISCUSSION TITLE
Unable to access through cloudflare tunnel

### DISCUSSION THREAD

**Discussion by fcassata** [docmost/docmost#208] [Help / Questions] [answered]
I get a blank page when I point a subdomain to the docker ip and port for docmost. Not sure what the issue is, but I believe it's a me-problem. Help would be greatly appreciated :)

**Comment by Philipinho** [comment]
What do you get when you visit the `subdomain_url.com/api` ?

**Reply by fcassata** [reply]
Thanks for the response! I get a 200

**Reply by fcassata** [reply]
In the blank page source, I have this line, not sure if relevant:

   <script>window.CONFIG={"ENV":"production","APP_URL":"https://mysubdomain.mydomain.com","IS_CLOUD":false};</script>

**Reply by Philipinho** [reply]
Check your browser console for any errors.

**Reply by fcassata** [reply]
One 415 there
<img width="281" alt="Screenshot 2024-08-27 at 00 26 03" src="https://github.com/user-attachments/assets/8972f7d2-9ea6-4402-9128-68ba949f1994">

**Reply by Philipinho** [reply]
Add the domain to the `APP_URL` environment variable.

edit: seems you have it already, judging by your previous post.
I will have a deeper look.

**Reply by fcassata** [reply]
It was already there, https://subdomain.domain.tld

**Reply by fcassata** [reply]
> edit: seems you have it already, judging by your previous post. I will have a deeper look.

Thanks, let me know if I can provide any further information!

**Comment by fcassata** [comment, marked as answer]
I was one version behind on cloudflared, after updating all is fine. Sorry about the confusion, and thanks for your work!
